config/coordinator.go

```diff
@@ -104,7 +104,7 @@ func (c *Coordinator) loadFromFile() error {
 	c.config = conf
 	c.configSuccessMetric.Set(1)
 	c.configSuccessTimeMetric.Set(float64(time.Now().Unix()))
-	hash := md5HashAsMetricValue(plainConfig)
+	hash := c.config.MD5()
```

Comment:
Since it's in the `config` package, `Coordinator` can access `conf.original` so this line doesn't have to change much (eg `hash := md5HashAsMetricValue(conf.original)`) and we don't need to introduce `Config.MD5()`.

Reply:
ok, I will change it.